src/Pages/ExerciseAndMuscleOverview.js

```diff
@@ -0,0 +1,121 @@
+import EntryMuscels from '../Components/ExercisesAndMuscels/EntryMuscels';
+import { useState } from 'react';
+import Card from '../Components/ExercisesAndMuscels/Card';
+import styled from 'styled-components';
+import NavEntryMuscels from '../Components/ExercisesAndMuscels/NavEntryMuscles';
+import BacktoTopButton from '../Components/BacktoTopButton';
+
+export default function ExerciseAndMuscleOverview({
+  sholder,
+  arms,
+  legs,
+  abs,
+  chest,
+  back,
+  images,
+}) {
+  const [activeMuscle, setActiveMuscle] = useState(null);
+
+  function handleActiveMuscle(name) {
+    setActiveMuscle(name);
+  }
+
+  return (
+    <Background>
+      {!activeMuscle ? (
+        <EntryMuscels handleActiveMuscle={handleActiveMuscle} />
```

Comment:
i think we can just send down "setActiveMuscle" as a prop instead of "handleActiveMuscle", which
becomes obsolete